src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Bindings/InnerApi.cs

```diff
@@ -0,0 +1,23 @@
+﻿using TechTalk.SpecFlow;
+
+namespace SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests.Bindings
+{
+    [Binding]
+    [Scope(Tag = "innerApi")]
+    public class InnerApi
+    {
+        private readonly TestContext _context;
+
+        public InnerApi(TestContext context)
+        {
+            _context = context;
+        }
+
+        [BeforeScenario(Order = 1)]
+        public void Initialise()
+        {
+            if(_context.InnerApi == null)
+                _context.InnerApi = new MockApi();
```

Comment:
It does't matter one bit, but there's funky new(ish) syntax:

    public void Initialise()
    {
        _context.InnterApi ??= new MockApi();
    }
`

Reply:
I like it